### PR TITLE
Release management: utility script to generate headers with various options (in place, etc) #534

### PR DESCRIPTION
**Pull request by vingar**

### CODE REVIEW

File: tools/add_header [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Authors:
+'''
+        for author in authors:
+            if int(authors[author]['min']) == int(authors[author]['max']):
+                header += '# - %(name)s, %(mail)s, %(min)s\n' % authors[author]
```
Comment (mlassnig) [inline tools/add_header]: i propose the RFC-822 style with brackets for emails

 %(name)s <%(mail)s>, %(min)s\n

Comment (bari12) [reply tools/add_header]: The list of mails is already filled with the brackets. So its fine.

Comment (bari12) [reply tools/add_header]: Line 51

Comment (mlassnig) [reply tools/add_header]: right, but the comma after the name needs to go

Comment (vingar) [reply tools/add_header]: The comma after the name is historically present in almost all files.
Example: lib/rucio/common/utils.py

Comment (mlassnig) [reply tools/add_header]: yep, but it should go ;-)

Comment (vingar) [reply tools/add_header]: let it go

**Review by mlassnig — APPROVED** [review]

**Pull request merged**